src/features/colinks/useDoWithCoLinksContract.ts

```diff
@@ -38,6 +38,11 @@ export const useDoWithCoLinksContract = () => {
       return;
     }
 
+    if (!authAccount) {
+      showError('Please connect your wallet');
```

Comment:
This should probably do something nicer like popup the wallet modal when they interact?

Reply:
Yeah, I don't know if we have a nice pattern for this right now.